src/software/backend/wifi_backend.cpp

```diff
@@ -31,8 +37,21 @@ WifiBackend::WifiBackend(std::shared_ptr<const BackendConfig> config)
 
 void WifiBackend::onValueReceived(TbotsProto::PrimitiveSet primitives)
 {
+    // check if estop has been set
+    if (estop_reader != nullptr && !estop_reader->isEstopPlay())
```

Comment:
isEstopPlay is confusing to me, can we call it isEstopOn or isEstopActivated?

Reply:
My fear is that talking about Estop being On/Activated is confused with Estop being Enabled (which is received from Robot Status message and is a precondition for being On/Activated). I am not sure how we have normally used Estop terms though

Reply:
ok it's fine as it is